lib/db_connection/holder.ex

```diff
@@ -438,4 +438,10 @@ defmodule DBConnection.Holder do
   defp cancel_deadline(deadline) do
     :erlang.cancel_timer(deadline, async: true, info: false)
   end
+
+  defp hash_pid(pid, interval_native) do
+    interval_ms = System.convert_time_unit(interval_native, :native, :millisecond)
```

Comment:
What if we don't convert from milliseconds to native on `DBConnection.disconnect_all`? This way we can skip this initial conversion here.

Reply:
Yep that's a good idea. I wasn't sure if it would be more confusing to have it stored in a different unit to the start timestamp, because I think it's stored as `{start, interval}`. But I can go ahead and do that if you think it makes more sense.

Reply:
Yeah, let's go ahead and change it there as well!